redisinsight/ui/src/pages/autodiscover-cloud/redis-cloud-subscriptions/RedisCloudSubscriptions/RedisCloudSubscriptions.tsx

```diff
@@ -106,7 +109,11 @@ const RedisCloudSubscriptions = ({
       subscriptions?.filter(
         (item: RedisCloudSubscription) =>
           item.name?.toLowerCase()?.indexOf(value) !== -1 ||
-          item.id?.toString()?.toLowerCase().indexOf(value) !== -1,
+          item.id?.toString()?.indexOf(value) !== -1 ||
```

Comment:
I'm wondering can't we add such util:
```
export const filterBySearchTerm = <T extends Record<string, unknown>>(
  items: T[],
  searchTerm: string,
  fields: (keyof T)[],
): T[] => {
  const term = searchTerm.toLowerCase()
  
  return items.filter((item) =>
    fields.some((field) => {
      const value = item[field]
      return String(value ?? '').toLowerCase().includes(term)
    }),
  )
}


or

export const matchesSearch = (
  fieldValue: unknown,
  searchTerm: string, // expected to be already lowercased
): boolean => String(fieldValue ?? '').toLowerCase().includes(searchTerm)
```
and then this syntax would be easier to be read and moreover we have it in the `RedisCloudDatabasesResult.tsx` as well.


Maybe not for this PR, but actually, why not 😸